motion/ruby_motion_query/event.rb

```diff
@@ -32,7 +32,7 @@ def initialize(sender, event, block)
 
     def handle_gesture_or_event(gesture)
       # Don't fire :long_press events twice.
-      return if @sdk_event_or_recognizer == VIEW_GESTURES[:long_press] && gesture.state == UIGestureRecognizerStateEnded
+      return if gesture.class == VIEW_GESTURES[:long_press] && gesture.state != UIGestureRecognizerStateBegan
```

Comment:
just curious, why the change from `@sdk_event_or_recognizer` to `gesture.class` ?  I think I am following whats going on here, but want to make sure :)